packages/block-editor/src/components/block-tools/block-selection-button.js

```diff
@@ -123,10 +123,11 @@ function BlockSelectionButton( { clientId, rootClientId } ) {
 
 	// Focus the breadcrumb in navigation mode.
 	useEffect( () => {
-		ref.current.focus();
-
-		speak( label );
-	}, [ label ] );
+		if ( editorMode === 'navigation' ) {
+			ref.current.focus();
+			speak( label );
+		}
+	}, [ label, editorMode ] );
```

Comment:
I'm unsure about this. It seems the core issue is that focusing this shouldn't automatically close the inserter. We're going to need to be able to have focus on the zoom-out mode toolbar and also stay in zoom-out mode. For example: 
- Insert a new pattern
- Click on the revealed pattern toolbar to move the pattern up/down.
- Pattern inserter is closed